widgetastic_patternfly.py

```diff
@@ -74,6 +97,12 @@ def __init__(self, parent, *text, **kwargs):
             # Join the kwargs
             self.locator_conditions = ' and '.join(
                 '@{}={}'.format(attr, quote(value)) for attr, value in kwargs.items())
+        classes = kwargs.pop('classes', [])
+        if classes:
+            self.locator_conditions += ' and '
+            self.locator_conditions += ' and '.join(
```

Comment:
if this is not nested ets use `+=  ''.join(' and contains...' ....)` - else the code reads like a triping point